lib/lte_link_control/lte_lc.c

```diff
@@ -1146,7 +1146,7 @@ int lte_lc_system_mode_get(enum lte_lc_system_mode *mode,
 			   enum lte_lc_system_mode_preference *preference)
 {
 	int err, mode_bitmask = 0;
-	int ltem_mode, nbiot_mode, gps_mode, mode_preference;
+	int ltem_mode, nbiot_mode, gps_mode, mode_preference = 0;
```

Comment:
I thought this only initializes `mode_preference` to `0`, and the other ones are left uninitialized